app/src/main/java/com/door43/translationstudio/service/ClientService.java

```diff
@@ -361,7 +361,7 @@ public void onClose() {
                                     String[] targetTranslationSlugs = translator.importArchive(file);
 
                                     TargetTranslationMigrator.migrateChunkChanges(translator, AppContext.getLibrary(), targetTranslationSlugs);
-
+                                    TargetTranslationMigrator.migrateFromUSXtoUSFM(targetTranslationSlugs);
```

Comment:
Just re-iterating. These manual calls to migrate can be made private and remove from here. The goal is to only have to call `translator.importArchive()` and all the magic happens underneath.